DOCS/man/options.rst

```diff
@@ -4256,6 +4265,10 @@ OSD
     See ``--sub-font-provider`` for details and accepted values. Note that
     unlike subtitles, OSD never uses embedded fonts from media files.
 
+``--osd-fonts-dir=<path>``
+    See ``--sub-fonts-dir`` for details.  Defaults to ``~~/fonts``.
+
+
```

Comment:
Sorry one really pointless nit but could you get rid of this extra newline?

Reply:
Sure thing!  :smile: